### PR TITLE
[OK-11409] No longer storing JCasC yaml files in the container and will rely on helm chart for them

### PR DESCRIPTION
**Pull request by ricardo-overhaul**

### CODE REVIEW

File: values.yaml [hunk 1]
```diff
@@ -0,0 +1,105 @@
+controller:
+  jenkinsUrl: jankins.aws.over-haul.com
```
Comment (jasonv-overhaul) [inline values.yaml:2]: Genius.  :-D

**Review by jasonv-overhaul — COMMENTED** [review]
@ricardo-overhaul a couple questions:
* How did you determine the format of the config files?
* In the configs you have credentials and unclassified. They both can rely on interpolation -- so what's the underlying difference?

**Comment by ricardo-overhaul** [comment]
@jasonv-overhaul 
- Format of config files https://github.com/jenkinsci/configuration-as-code-plugin#getting-started
- `unclassified` refers to `configuration-as-code`
image

**Review by tyler-overhaul — APPROVED** [review]
LGTM (Little Green Tiny Margerhitas) 👍

**Pull request merged**